pandas/util/_decorators.py

```diff
@@ -50,14 +53,15 @@ def deprecate(
         Version of pandas in which the method has been deprecated.
     alt_name : str, optional
         Name to use in preference of alternative.__name__.
-    klass : Warning, default FutureWarning
+    klass : Warning, optional
+        The warning class to use.
     stacklevel : int, default 2
     msg : str
         The message to display in the warning.
         Default is '{name} is deprecated. Use {alt_name} instead.'
     """
     alt_name = alt_name or alternative.__name__
-    klass = klass or FutureWarning
+    klass = klass or Pandas40DeprecationWarning
```

Comment:
I wonder if there should be a variable pointing to the current "active" deprecation warning (i.e. the next ``PandasXXDeprecationWarning`` that would still raise a ``DeprecationWarning``. Once we get to the last minor version before a major release, this variable would then be bumped to the next warning.), that we can use here.

I think it might be easy to forget to change this.